AzFilesHybrid/AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -3170,13 +3170,11 @@ function Test-Port445Connectivity
 
         if ($result.TcpTestSucceeded -eq $False)
         {
-            $message = "Unable to reach the storage account file endpoint." `
-                + "`n`tTo debug connectivity problems, please refer to the troubleshooting tool for Azure" `
-                + " Files mounting errors on Windows, " `
-                + " `n`t'AzFileDiagnostics.ps1'($($PSStyle.Foreground.BrightCyan)https://gallery.technet.microsoft.com/Troubleshooting-tool-for-a9fa1fe5$($PSStyle.Reset))." `
-                + " `n`tFor possible solutions please refer to" `
-                + " '$($PSStyle.Foreground.BrightCyan)https://aka.ms/azfiles/entra-port445$($PSStyle.Reset)'"
-            Write-Error -Message $message -ErrorAction Stop
+            $errMsg = "Unable to reach the storage account file endpoint." `
+            + "`n`tTo debug connectivity problems, please refer to the following," `
+            + "`n`t'AzFileDiagnostics.ps1'($($PSStyle.Foreground.BrightCyan)https://github.com/Azure-Samples/azure-files-samples/tree/master/AzFileDiagnostics/Windows$($PSStyle.Reset))'." `
+            + "`n`tFor possible solutions please refer to '$($PSStyle.Foreground.BrightCyan)https://aka.ms/azfiles/entra-port445$($PSStyle.Reset)'"
+            Write-TestingFailed -Message $errMsg -ErrorAction Stop

```

Comment:
I don't think the `-ErrorAction Stop` is doing anything here, we can remove it. To test this, you can try:

```powershell
try { Write-Error "test" -ErrorAction Stop } catch { Write-Host "caught write-error" }
try { Write-TestingFailed "test" -ErrorAction Stop } catch { Write-Host "caught write-testingfailed" }
```

![image](https://github.com/user-attachments/assets/ce4dad83-e602-49a7-82aa-7160d33e2d73)